led_v2_code

BaSSeL42 opened the pull request.

BaSSeL42 committed:
led_v2_code

MahmoudMatarawy committed:
add initial project files

MahmoudMatarawy committed:
update App

BaSSeL42 committed:
Code Modification & Documents

BaSSeL42 committed:
Team backlog

MahmoudMatarawy committed:
update app

BaSSeL42 committed:
Code Modification & Documents

BaSSeL42 committed:
Documents & video

Hala-Abdelkader commented on led_v2/code/APP/app.c:
```diff
@@ -0,0 +1,130 @@
+#include "led.h"
+#include "button.h"
+#include "app.h"
+#include "systick_Interface.h"
```
You shouldn't include drivers from MCAL layer in application

BaSSeL42 replied:
We resolved it

Hala-Abdelkader replied:
Great, well done.

MahmoudMatarawy committed:
fix bug

Include systick manager from the HAL layer instead of including systick driver from MCAL layer directly

BaSSeL42 committed:
MGPIO Modified
